tests/commands/test_geturl_utils.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from unittest import mock
+from tabcmd.commands.datasources_and_workbooks.get_url_command import *
+
+
+class GetURlTests(unittest.TestCase):
+
+    def test_evaluate_file_name_pdf(self):
+        mock_logger = mock.MagicMock()
+        filename = 'filename.pdf'
+        url = None
+        filetype = GetUrl.evaluate_file_name(mock_logger, filename, url)
+        assert filetype == 'pdf', filetype
+
+    def test_evaluate_file_name_url(self):
+        mock_logger = mock.MagicMock()
+        filename = None
+        url = 'long-url-stuff-goes-here.pdf'
+        filetype = GetUrl.evaluate_file_name(mock_logger, filename, url)
+        assert filetype == 'pdf', filetype
+
+    """
```

Comment:
Are these the remaining tests we should write?

Reply:
it's a list of all the methods in the file, we should have a unit test that at least executes each of them